app/code/community/Oyst/OneClick/Helper/ZipMap.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * This file is part of Oyst_OneClick for Magento.
+ *
+ * @license http://www.apache.org/licenses/LICENSE-2.0 Apache License, Version 2.0
+ * @author Oyst <plugin@oyst.com> <@oyst>
+ * @category Oyst
+ * @package Oyst_OneClick
+ * @copyright Copyright (c) 2017 Oyst (http://www.oyst.com)
+ */
+
+/**
+ * ZipMap class
+ */
+class Oyst_OneClick_Helper_ZipMap
+{
+    const ZIP_MAP = array(
```

Comment:
In PHP 5.3 https://travis-ci.org/oystparis/oyst-1click-magento/jobs/345208028#L963

So could you use a property.